### PR TITLE
app background styling

### LINKED ISSUE #24: App background styling
**Is your feature request related to a problem? Please describe.**
App background is not styled

**Describe the solution you'd like**
Style App background

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Added gradient to app background
2. Aligned headings
3. Adjusted logo

## Purpose
Enhance user experience through UI.

## Approach
scss

Closes #24

### CODE REVIEW

File: movie-app/src/App.js [hunk 1]
```diff
@@ -2,26 +2,37 @@ import "./scss/main.scss";
...
-import { Home } from './body/Home';
-
+import Searched from "./body/Searched";
+import { Home } from "./body/Home";
 
 function App() {
   const [movie, setMovie] = useState({});
   const [movieSearch, setMovieSearch] = useState();
-  const [page, setPage] = useState(1)
+  const [page, setPage] = useState(1);
```
Comment (BlackCubes) [inline movie-app/src/App.js:11]: Nice semi-quote!

File: movie-app/src/App.js [hunk 2]
```diff
@@ -2,26 +2,37 @@ import "./scss/main.scss";
...
+            movies={movie.Search}
+            setPage={setPage}
+            page={page}
+          />
+        ) : (
+          <h2 className="error-text"> Please enter a valid movie title </h2>
+        )
+      ) : (
+        <Home />
+      )}
```
Comment (BlackCubes) [inline movie-app/src/App.js:33]: Looks easier to read. Noice!

File: movie-app/src/header/NavBar.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ const NavBar = ({ movieSearch, setMovieSearch }) => {
     <div className="header">
       <nav className="navigation">
         <div className="logo-container">
-          <img src={"./logo.svg"} alt="ToadTv logo" className="logo" />
+          <img src={"logo.png"} alt="ToadTv logo" className="logo"  />
```
Comment (BlackCubes) [inline movie-app/src/header/NavBar.js:23]: Looks like it works (I think?)

File: movie-app/src/scss/components/body/_search_results.scss [hunk 4]
```diff
@@ -1,57 +1,53 @@
...
-    grid-row-gap: 20px;
+  width: 100vw;
+  display: grid;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-left: 5%;
+  padding-right: 5%;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
```
Comment (BlackCubes) [inline movie-app/src/scss/components/body/_search_results.scss:10]: CSS takes a long time to make it look beautiful, but this looks like a masterpiece

**Review by BlackCubes — APPROVED** [review]
Everything looks amazing. Beautiful job!

File: movie-app/src/header/NavBar.js (hunk 3 above)
Comment (krferrell) [reply movie-app/src/header/NavBar.js:23]: Yeah, it does lol

**Pull request merged**